renovate.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
what is renovate.json?

Reply:
this is something I will add as integration so we get packages updated frequently. take a look at my projects. you can see what it does.

Reply:
I enabled dependabot for this repo, I think it's a more popular alternative. Please remove this file.